data-products/src/recommendation_api/sql/labeled_collection_stories.sql

```diff
@@ -0,0 +1,31 @@
+WITH collections_with_label_expanded AS (
```

Comment:
don't need to do this, but this query might be slightly more obvious:

```
with labeled_stories as (
    SELECT
        slug, language, l.value:name as label, JSON_EXTRACT_PATH_TEXT(s.value, 'url') as url
    FROM
        "ANALYTICS"."DBT"."COLLECTIONS",
        LATERAL FLATTEN(INPUT => labels) as l,
        lateral flatten(INPUT => stories) as s
    where language = 'EN'
    and label = 'idea'
)
select 
    aci.approved_corpus_item_external_id as "ID",
    aci.topic as "TOPIC",
    aci.publisher as "PUBLISHER"
from analytics.dbt.approved_corpus_items as aci
inner join labeled_stories as ls on ls.url = aci.url
```

Reply:
Using it!